configs/README.md

```diff
@@ -0,0 +1,264 @@
+* `dhcpv4` - REQUIRED. `false` for static interfaces
+* `ipv4_addr` - REQUIRED. IP address to assign to the interface
+* `ipv4_mask` - REQUIRED. IP mask to use for the subnet
+* `hw_addr` - MAC address to use for the interface
+
+### DHCP interface:
+
+```
+- if_name: eth1
+  existing: override
+  dhcpv4: false
+  ipv4_addr: 10.0.103.103
+  ipv4_mask: 24
+  hw_addr: "{{ARCH_ETHERNET_MAC}}"
+```
+
+Breakdown:
+* `if_name` - REQUIRED. Name of the interface Maestro will modify
+* `existing` - OPTIONAL. Tells Maestro to `replace` or `override` the existing saved interface
+* `dhcpv4` - REQUIRED. `false` for static interfaces
+* `ipv4_addr` - REQUIRED. IP address to assign to the interface
+* `ipv4_mask` - REQUIRED. IP mask to use for the subnet
+* `hw_addr` - MAC address to use for the interface
+
+## DeviceDB
+
+Only necessary if you want Maestro to connect to a deviceDB server:
+
+```
+devicedb_conn_config:
+    devicedb_uri: "https://localhost:9090"
+    devicedb_prefix: "vagrant"
+    devicedb_bucket: "lww"
+    relay_id: "2462a03b129928209ce396f"
+    ca_chain: "~/.ssl_certs/myCA.pem"
+```
+
+Breakdown:
+* `devicedb_uri` - REQUIRED. URL of the deviceDB edge instance. NOT the deviceDB cloud URL
+* `devicedb_prefix` - REQUIRED. Table within devicedb to put data into
+* `devicedb_bucket` - REQUIRED. Bucket within the table specified above.
+* `relay_id` - REQUIRED. Unique identifier for the gateway. Usually the gateway ID specified by the cloud
+* `ca_chain` - REQUIRED. Location of the root CA certificate that the deviceDB cloud instance is setup with.
+
+## Logging
+
+### SysLog
+
+To enable syslog, add to your `maestro.config`:
+
+```
+sysLogSocket: /run/systemd/journal/syslog
+```
+
+Where `/run/systemd/journal/syslog` is the path to your syslog file.
+
+To inject a log into a log target, you can use syslog:
+
+```
+echo "test err message" | systemd-cat -p err
+```
+
+Where the contents of the `echo` is the message, and `err` is the log level. Note that you must have the log level you specify match what is in your log target filters list otherwise it will not show up.
+
+### Targets
+
+```
+targets:
+    - <target 1> # Outlined below
+    - <target 2> # Outlined below
+    - <target ...> # Outlined below
+```
+
+Breakdown:
+* `name` - Unique identifier for the log target. `toCloud` is a special name for sending data to the cloud
+* `file` - Required for log targets destined to a file. This is the output location for said file.
+* `format_time` - REQUIRED. Used to specify the time format in the output logs
+* `delim` - OPTIONAL. Used to specify the deliminator between logs
+* `filters` - REQUIRED. Used to specify what level of logs make it to the output log
+    * `warn`
+    * `info` or `success`. They behave the same
+    * `error`
+    * `debug`
+    * `all`
+* `rotate` - Optional for log targets destined to a file. Determines how to rotate between X amount of files
```

Comment:
I'd reword this, the current wording makes it sound like it's optional when logging to a file, but mandatory otherwise.
Maybe this: "Optional; only used for log targets destined to a file."